nuxeo-drive-core/src/main/java/org/nuxeo/drive/adapter/impl/AbstractDocumentBackedFileSystemItem.java

```diff
@@ -57,7 +57,7 @@
 
     private static final long serialVersionUID = 1L;
 
-    private static final Log log = LogFactory.getLog(AbstractDocumentBackedFileSystemItem.class);
+    private static final Logger log = LogManager.getLogger();
```

Comment:
Just found that this method won't be compatible with java 11 due to usage of `sun.reflect.Reflection.getCallerClass`, we should use version which takes a class.